src/cirrus/docs.py

```diff
@@ -34,13 +34,18 @@ def build_parser(argslist):
               'to run the Sphinx make command. Default: clean html'))
     build_command.set_defaults(make=[])
 
-    pack_command = subparsers.add_parser(
+    subparsers.add_parser(
         'pack',
         help='Package documentation as a tarball')
 
     publish_command = subparsers.add_parser(
         'publish',
         help='Publish documentation as specified in cirrus.conf')
+    publish_command.add_argument(
+        '--test',
+        action='store_true',
```

Comment:
might want to add default=False here to enforce that it is only set to True if the flag is provided. I think it defaults to None, but would make it explicit